Library/Homebrew/cmd/info.rb

```diff
@@ -401,6 +411,62 @@ def info_cask(cask)
 
         Cask::Info.info(cask, args:)
       end
+
+      sig { params(title: String, items: T::Array[T::Hash[Symbol, T.any(String, Integer)]]).void }
```

Comment:
Would be nice to use a Sorbet `T::Struct` for the `T::Array` value to avoid needing to do the `T.cast`s below.